src/main/java/fr/lunatech/timekeeper/services/UserEventService.java

```diff
@@ -90,6 +114,21 @@ public Long createOrUpdateFromEventTemplate(EventTemplate eventTemplate,
         return updated;
     }
 
+    /**
+     * TODO add uservent rules
```

Comment:
Can you create a ticket to track this TODO on Jira please ? Thanks

Reply:
TK-472